test/docker_container_test_case.py

```diff
@@ -90,13 +95,26 @@ def _create_container(cls, image: str, container_port: int, **kwargs) -> Netloc:
                 log.info('Launched sibling container %s from image %s, listening on %s:%i',
                          container.name, image, container_ip, container_port)
             else:
+                retries = 0
                 ports = network_settings['Ports']
+                while len(ports[f'{container_port}/tcp']) < 1:
+                    if (datetime.now() - time_it).seconds > 3:
+                        # Wait for the published ports of the container that's
+                        # supposedly running, otherwise giveup (let it fail).
+                        log.error('Unreachable TCP port %s for container %s',
+                                  container_port, container.name)
+                        break
+                    time.sleep(.33)
+                    container_info = cls._docker.api.inspect_container(container.id)
+                    ports = container_info['NetworkSettings']['Ports']
+                    retries += 1
+                milisecs = (datetime.now() - time_it).microseconds / 1000
```

Comment:
"milliseconds" has 2 L's
```suggestion
                millisecs = (datetime.now() - time_it).microseconds / 1000
```

Reply:
I don't think this is doing what you want:

```
>>> from datetime import timedelta
>>> timedelta(seconds=3).microseconds
0
```

Reply:
ty